analyzer/codechecker_analyzer/buildlog/log_parser.py

```diff
@@ -497,47 +524,26 @@ def get_compiler_standard(compiler, language):
         return standard
 
     @staticmethod
-    def load_compiler_info(filename, compiler):
-        """Load compiler information from a file."""
-        contents = load_json_or_empty(filename, {})
-        compiler_info = contents.get(compiler)
-        if compiler_info is None:
-            LOG.error("Could not find compiler %s in file %s",
-                      compiler, filename)
-            return
+    def dump_compiler_info(filename):
```

Comment:
The parameter name is missleading. It's not a file name but a file path.

```suggestion
    def dump_compiler_info(file_path: str):
```